Alexandr Antsiferov/lesson7/src/app/redusers/userReduser.js

```diff
@@ -0,0 +1,15 @@
+export default function reducer(
+  state = {
+    user: {}
+  },
+  action) {
+  switch (action.type) {
+    case "FETCH_USER_FULFILLED": {
+      return { ...state, user: action.payload };
+    }
+    case "RESET_USER_FULFILLED": {
```

Comment:
```suggestion
    case "RESET_USER": {
```
`RESET_USER` - это тип `action'a` (свойство объекта), а данный action (объект) не несет с собой в качестве другого свойства (`payload`) какой-либо `promise`, поэтому `promise-middleware` не изменит `RESET_USER` на `RESET_USER_FULFILLED`

Reply:
Спасибо. Странно что я сразу не заметил, в консоли ж написано)